draft-ietf-quic-qpack.md

```diff
@@ -1153,7 +1153,7 @@ represented as a 4-bit prefix string literal, then the value, represented as an
 
 #  Configuration
 
-QPACK defines two settings which are included in the HTTP/3 SETTINGS frame.
+QPACK defines two settings that are included in the HTTP/3 SETTINGS frame:
```

Comment:
Seems a bit strange, since it seems to imply that these SETTINGS are always present.

Would this be better:

"QPACK defines two settings for the HTTP/3 SETTINGS frame" ?